src/sceneHandler/index.ts

```diff
@@ -8,6 +8,7 @@ import {
 import { DataModal, GameScore } from "@data";
 import { Debugger } from "@common";
 import {
+  SCENE_NAME_LOADING,
```

Comment:
Add todo to refactor later.

From a quick scan, it looks like the `loading-scene` is really a transition between scenes.
We can reactor this to a transition scene in the scene-handler that would show when changing scenes.

Reply:
The variable SCENE_NAME_LOADING has been renamed to LOADING_TRANSITION.